doc/modules/clustering.rst

```diff
@@ -278,6 +285,67 @@ small, as shown in the example and cited reference.
    D. Sculley, *Proceedings of the 19th international conference on World
    wide web* (2010)
 
+.. _k_medoids:
+
+K-Medoids
+=========
+
+
+:class:`KMedoids` is related to the :class:`KMeans` algorithm. While
+:class:`KMeans` tries to minimize the within cluster sum-of-squares,
+:class:`KMedoids` tries to minimize the sum-of-distances between points labeled
```

Comment:
"the sum of distances between each point and the medoid of its cluster."

Perhaps something like: "The medoid is a data point (unlike the centroid) which has least total distance to the other members of its cluster."